src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -123,6 +123,27 @@ struct DistributedSearchTreeImpl
                     typename View::non_const_type imports);
 };
 
+namespace internal
```

Comment:
Is this the right place for it? Or should it go to Kokkos utilities file?

Reply:
Also, do we want to test that somewhere?

Reply:
@masterleinad So that you know, the convention in the Search package of DTK use to be `namespace Details { ... }`.  I am not asking you to change, it is just a FYI.  `internal` is standard enough and it will be easy to grep for it.

@aprokop It is the right place for now as it is only used here.

About testing, I guess we could add a test with a view that has compile-time dimension in the  `create_layout_right_mirror_view` test case and it would be good enough.